### PR TITLE
167 confirm pending

### PR DESCRIPTION
**Pull request by dwalintukan**
#167 #166 
- added `includeRoundBets` param for `events` query to return round balances

### CODE REVIEW

File: src/utils/web3-utils.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const { isNull } = require('lodash');
+const web3 = require('../web3');
```
Comment (henryzbw) [inline src/utils/web3-utils.js]: const  { web3 } = require('../web3');

File: src/utils/web3-utils.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const TransactionReceipt = require('../models/tx-receipt');
+
+/**
+ * Gets the pending tx and conforms it to the TransactionReceipt model.
+ * @param {string} txid Transaction ID
+ * @return {TransactionReceipt} Transaction receipt
+ */
+const getTransaction = async (txid) => {
+  const tx = web3().eth.getTransaction(txid);
```
Comment (henryzbw) [inline src/utils/web3-utils.js]: need to await

File: src/utils/web3-utils.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  return new TransactionReceipt({
+    transactionHash: tx.hash,
+    blockHash: tx.blockHash,
+    blockNumber: tx.blockNumber,
```
Comment (henryzbw) [inline src/utils/web3-utils.js]: tx.blockNumber is null when i get it
TransactionReceipt validation at this part has problem. see the tx returned from `web3().eth.getTransaction(txid);` 
image

File: src/utils/web3-utils.js (hunk 1 above)
Comment (dwalintukan) [reply src/utils/web3-utils.js]: fixed

File: src/utils/web3-utils.js (hunk 2 above)
Comment (dwalintukan) [reply src/utils/web3-utils.js]: fixed

File: src/utils/web3-utils.js (hunk 3 above)
Comment (dwalintukan) [reply src/utils/web3-utils.js]: fixed

**Pull request merged**